packages/react-icons/src/createIcon.tsx

```diff
@@ -66,13 +67,11 @@ export function createIcon({
 
       const hasTitle = Boolean(title);
       const heightWidth = getSize(size);
-      const baseAlign = -0.125 * Number.parseFloat(heightWidth);
-      const style = noVerticalAlign ? null : { verticalAlign: `${baseAlign}em` };
       const viewBox = [xOffset, yOffset, width, height].join(' ');
 
       return (
         <svg
-          style={style}
+          className={noVerticalAlign ? undefined : css('pf-svg-vertical-align')}
```

Comment:
Change `css('pf-svg-vertical-align')` to `'pf-svg-vertical-align'`

Reply:
ok